src/swap/types.ts

```diff
@@ -219,6 +219,15 @@ export type SwapReact = {
   title?: string; // Title for the Swap component. (default: "Swap")
 };
 
+/**
+ * Note: exported as public Type
+ */
+export type SwapSettingsReact = {
+  className?: string; // Optional className override for top div element.
+  icon: ReactNode; // Icon for the settings button
+  title: string; // Title text for the settings button
```

Comment:
I think we want to align over time to only allow `text` update in two way, either:
- `text` prop
or
- `children` prop

and case by case we decide which one is best.

Reply:
Update to use the parameter name `text` instead of `title`.